src/scss/_functions.scss

```diff
@@ -26,6 +26,23 @@
 	@return false;
 }
 
+/// Check if a font variant has been included
+/// in your project with the `oFonts` mixin.
+///
+/// @access public
+/// @param {String} $family - one of $_o-fonts-families
+/// @param {String} $weight [regular] - The font weight.
+/// @param {String} $style [normal] - The font style.
+/// @return {Bool}
+/// @see oFonts To include fonts
+@function oFontsVariantIncluded($family, $weight, $style) {
```

Comment:
Do we need to add this to the README documentation?

Reply:
yes 👍